app/services/cards_parser_service.rb

```diff
@@ -0,0 +1,35 @@
+class CardsParserService
+  class << self
+    def schedule_parsing(user_id, params)
+      block_id = params[:block_id]
+      return false unless block_id && user_id
+
+      get_text(params)
+
+      return false unless @original.size == @translated.size && @original.any?
+      return false if @original[0] == @translated[0]
+
+      Card.delay.create_from_html(user_id, block_id, @original, @translated)
+      true
+
+    rescue Exception
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?